### PR TITLE
ci(connect): skip passphrase test for webextension

### PR DESCRIPTION
**Pull request by martykan**
## Description

As mentioned in the Code improvements session, we want to improve the reliability of tests, so I'm disabling one of the very flaky E2E tests in Connect - passphrase in webextension.

### CODE REVIEW

File: .github/workflows/connect-dev-release-test.yml [hunk 1]
```diff
@@ -80,7 +80,8 @@ jobs:
     with:
       test-name: passphrase.test
       DEV_SERVER_HOSTNAME: dev.suite.sldev.cz
-      run-webextension: true
+      # currently disable due to instability
+      # run-webextension: true
```
Comment (karliatto) [inline .github/workflows/connect-dev-release-test.yml]: Why don't we run it only on nightly runs instead of totally disabling it?

Comment (martykan) [reply .github/workflows/connect-dev-release-test.yml]: Ok I think that's a reasonable compromise, but in the end we want to fix the cause of the flakiness

This should do it, right?
```
${{ github.event_name == 'schedule' }}
```

Comment (karliatto) [reply .github/workflows/connect-dev-release-test.yml]: Of course, we need to fix the cause of flakiness but I would feel safer if we see those test running at least once a day.

Yes, I think `${{ github.event_name == 'schedule' }}` should work as expected there.

**Review by karliatto — APPROVED** [review]
:+1: 

Having it running nightly works as a reminder that we have fix it.

**Pull request merged**